lib/server.js

```diff
@@ -0,0 +1,19 @@
+import { Server } from 'hapi'
+
+export default (port) => {
+  const server = new Server()
+
+  server.connection({port: port})
+
+  server.start(() => {
+    console.log(`Running at http://localhost:${port}`)
+  })
```

Comment:
Also, I'm totally totally new to this....if I'm not doing something the way you would, it's probably not because I think there's a downside, but just because I don't know the option exists :cry: